app/javascript/data/advancedFilterOptions.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 export const advancedFilterOptions = [
   /* BOOLEAN FILTER OPTIONS */
   {
-    name: 'continous-exposure',
+    name: 'continuous-exposure',
```

Comment:
oof we're going to have to do a migration to fix the saved filters already in production @sgober @tstrass just tagging you for awareness

Reply:
I think this does need to be fixed at some point but maybe just create a separate ticket?

Reply:
@sgober Is your recommendation that we change the filter option back to the misspelling (and add a comment that changing it would require creating a migration for saved filters in production) in this PR?

Reply:
Well the migration needs to be done with this change, so I think it's most appropriate that we create a sara jira ticket to fix the spelling and add the migration because it's not relevant to this PR.  @tstrass thoughts?

Reply:
@sgober I'm on board with fixing this in a separate PR

Reply:
We will revert this completely (without any comment to prevent any future git-merge annoyances). I will however add a JIRA ticket and comment it here.

Reply:
[JIRA Ticket Created - SARAALERT-1603](https://tracker.codev.mitre.org/browse/SARAALERT-1603).

[Commit with spelling reversion - e02ae3b2f9](https://github.com/SaraAlert/SaraAlert/pull/1025/commits/e02ae3b2f97812ff4a59d55d94f365b634cea83e).